security-acceptance-tests/src/test/resources/cucumber/SimpleSecurityTest.feature

```diff
@@ -0,0 +1,10 @@
+@security-demo
+Feature: Demonstrate the Security test framework
+
+  Scenario: Simple security test
+    Given I am on the Google search page
+    When I search for "OpenCredo"
+    Then the site "www.opencredo.com" should be present in the results
+    And the number of risks per category should not be greater than
```

Comment:
This doesn't really mean a great deal to me. Made a similar point on the blog post, would like this to be more meaningful. Maybe just saying "security risks" would help? Also, what's a category?